Extensions/mutualchecker.js

```diff
@@ -100,6 +121,17 @@ XKit.extensions.mutualchecker = new Object({
 		}
 	},
 
+	plain_check: function(json_obj, $link) {
```

Comment:
what does `plain_check` mean? could you add some inline documentation here to make this clearer? [JSDocs](https://github.com/shri/JSDoc-Style-Guide) can be super helpful to people trying to contribute to your code 😄

Reply:
> prefer a descriptive function name to inline documentation

100% that this is generally the best first step, but overall i think it's a great one-two punch for any shared codebase to complement descriptive feature names with detailed inline documentation. there are many times when you just can't make a function or variable name descriptive enough, like the always-around `options` object variable, which is `json_obj` here

Reply:
sure, but in this case that's because the function is taking a opaque and poorly specified object when it's only using one of its variables—this should just be `group_blog_fallback(username, $link)`. json_obj should probably be named something like popover_metadata, and we should explicitly destructure the variables we expect from it when we parse it from the DOM. (`const {name, following, customizable} = JSON.parse(popover_metadata)`). Inline documentation in this case would obscure the more fundamental encapsulation problem.

inline documentation is bad because either 1) you're making up for illegible code by half-heartedly explaining it, and any change is going to put your comment out of date, or 2) you're justifying design decisions that should be written up in a more long-form way and live in the pull request description or commit message—because they're also a record of a particular place and time and should be tied to their specific context, not living in the codebase forever.